Assoumpta/MyDiary
# all files of views are commited

What does this PR do?
This PR shows the page where users will be able to view the entries and their published date.

Description of Task to be completed?
Design a page that helps the user to enter new entry.
Design a page that helps a user to view the entered entries.

How should this be manually tested?
clone a repository.
git checkout ft-view-168055248

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
Create a repository.
As a user I should be able to view the entries  of my dairy

Screenshots (if appropriate)
![viewentry](https://user-images.githubusercontent.com/54316466/64231388-5c4cb480-cea4-11e9-940c-0048cd05ef7b.PNG)
